### ISSUE TITLE
Compatibility with webpack 5?

### ISSUE THREAD

**Issue by LouisTrezzini** [getsentry/sentry-webpack-plugin#237]
Hello

We are trying to use this plugin with webpack 5 but we run into this error:
```
$ webpack --config webpack.config.web.js --mode production
[webpack-cli] [31mError: Unsupported chunk loading type /var/lib/jenkins/workspace/Web Client Build/node_modules/@sentry/webpack-plugin/src/sentry-webpack.module.js.
Plugins which provide custom chunk loading types must call EnableChunkLoadingPlugin.setEnabled(compiler, type) to disable this error.
    at EnableChunkLoadingPlugin.apply (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack/lib/javascript/EnableChunkLoadingPlugin.js:104:12)
    at WebpackOptionsApply.process (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack/lib/WebpackOptionsApply.js:139:40)
    at createCompiler (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack/lib/webpack.js:78:28)
    at create (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack/lib/webpack.js:115:15)
    at webpack (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack/lib/webpack.js:139:31)
    at f (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack/lib/index.js:35:15)
    at WebpackCLI.createCompiler (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack-cli/lib/webpack-cli.js:219:24)
    at WebpackCLI.run (/var/lib/jenkins/workspace/Web Client Build/node_modules/webpack-cli/lib/webpack-cli.js:248:31)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)[39m
error Command failed with exit code 2.
```

Is webpack 5 supported? Do you have any plan to support it or do you know a workaround? 

Thanks!

**Comment by LouisTrezzini** [comment]
Hello @dcramer, 

Can we expect an update on this issue? Or should we completely move away from `sentry-webpack-plugin`? 
This is blocking our migration to webpack 5

Thank you

**Comment by donlion** [comment]
@dcramer It has become an upgrade blocker for us too now.
Any news?

**Comment by kamilogorek** [comment]
Repro case would be a great start, as I'm not sure what your config is, and the basic Webpack setup is working as expected. Thanks.

![image](https://user-images.githubusercontent.com/1523305/100205369-53129200-2f05-11eb-9e44-4c26ddd70ac2.png)

**Comment by kamilogorek** [comment]
Released as `1.14.0`

**Comment by LouisTrezzini** [comment]
Thank you!
